File: Patterns/python/pattern3.py
# n = 4
# 1234
# 1234
# 1234
# 1234

n = int(input())
row = 1
while row <= n:
    col = 1
    while col <= n:
        print(col, end="")
        col += 1
    row += 1
    print("")
    

File: Patterns/js/pattern3.js
function pattern(n) {
  let ans = "";
  let row = 1;
  let val = 1;

  while (row <= n) {
    let space = n - row;
    let s = 1;
    while (s <= space) {
      ans += " ";
      s++;
    }
    let col = 1;
    while (col <= row) {
      ans += val;
      val++;
      col++;
    }

    ans += "\n";
    row++;
  }

  console.log(ans);
}

pattern(4);
